contracts/contracts/ValidatorAuction.sol

```diff
@@ -13,7 +13,8 @@ contract ValidatorAuction is Ownable {
 
     uint public startTime;
     uint public closeTime;
-    uint public constant AUCTION_DURATION = 14 days;
+    uint public auctionDurationInDays;
+    uint public startPrice;
     uint public constant NUMBER_OF_PARTICIPANTS = 123;
```

Comment:
Might as well parametrise the number of participants

Reply:
yeah, but I thought we dont need it really. Do you think I should do it?

Reply:
okay